qrm/qrm_pgx.go

```diff
@@ -8,9 +8,63 @@ import (
 	"reflect"
 )
 
-// QueryablePGX interface for pgx Query method
+// QueryablePGX interface for pgx QueryContext method
 type QueryablePGX interface {
-	Query(ctx context.Context, sql string, args ...any) (pgx.Rows, error)
+	QueryContext(ctx context.Context, query string, args ...any) (pgx.Rows, error)
```

Comment:
Does pgx have `QueryContext` method?

Reply:
This interface is for `pgx` types. Since there is no `QueryContext` method in `pgx`, the build is failing:

```sh
=== Errors
tests/postgres/pgx_test.go:34:49: cannot use pgxDB (variable of type *pgx.Conn) as qrm.QueryablePGX value in argument to qrm.QueryPGX: *pgx.Conn does not implement qrm.QueryablePGX (missing method QueryContext)
```